util/network-devp2p/src/disk.rs

```diff
@@ -0,0 +1,94 @@
+use log::*;
```

Comment:
Module docs and license header?

Reply:
Not including the license header here since it's a new post-Parity file and as discussed, it will just be in a separate contributions file.

I'm not sure what to include in module docs at this point since `DiskEntity` is documented and `save` and `load` are pretty much self-descriptive.

Reply:
> I'm not sure what to include in module docs at this point

"This module is a utility for when you need to persist some small amount of data on disk, e.g. a secret or a snippet of user configuration. Implement Blabla for your type and call blabla to retrieve it again. It is/is not thread safe and should/should not be used when X Y Z or when performance is critical" etc?